RELEASE/scripts/sl_ascend/sl_combat.ash

```diff
@@ -583,6 +583,11 @@ string sl_combatHandler(int round, string opp, string text)
 		}
 	}
 
+	if(enemy.to_string() == "the invader" && sl_have_skill($skill[Weapon of the Pastalord]))
+	{
+		return "skill " + $skill[Weapon of the Pastalord];
```

Comment:
`return useSkill($skill[Weapon of the Pastalord], false);` is preferable.

Reply:
👍